report/testData/httpmockgerrit/fake_commits_latest.json

```diff
@@ -0,0 +1,24 @@
+)]}'
+{
+"commit": "06d492aade888ab8698aad35476286b7b555c961",
+"parents": [
+{
+"commit": "929e72ca90deac4784bbe451caf10faa5b256ebe",
+"subject": "message/pipeline: detect unknown keys"
+}
+],
+"author": {
+"name": "Jeremy Faller",
+"email": "jeremy@golang.org",
```

Comment:
For 'fake' data, this email looks pretty real. Perhaps `jeremy@example.com`

While I am confident the current code will never send this person an email, imo it would be better to truly make fake data to ensure there are never any accidental emails with this testing data.